ui/app/helpers/utils/util.js

```diff
@@ -407,3 +407,35 @@ export function constructTxParams ({ sendToken, data, to, amount, from, gas, gas
   }
   return addHexPrefixToObjectValues(txParams)
 }
+
+/**
+ * Makes a JSON RPC request to the given URL, with the given RPC method and params.
+ *
+ * @param {string} rpcUrl - The RPC endpoint URL to target.
+ * @param {string} rpcMethod - The RPC method to request.
+ * @param {Array<unknown>} rpcParams - The RPC method params.
```

Comment:
This appears to be optional:
```suggestion
 * @param {Array<unknown>} [rpcParams] - The RPC method params.
```

Reply:
4f22100